client/src/templates/root.njk

```diff
@@ -27,7 +27,7 @@
   </div>
 
   <footer>
-    PolyChampions website footer. Created by Luna and Artemis. Stats brought to you with help from Legorooj, Nelluk, and the PolyChampions stats team. Graphics done by Newt.<br><br>Join now on Discord!
+    PolyChampions website footer. Created by Luna and Artemis. Stats brought to you with help from Legorooj, Nelluk, and the PolyChampions stats team. Graphics done by Newt.<br><br> <a href="https://discord.gg/6cvhe4D">Join now on Discord!</a>
```

Comment:
```suggestion
    PolyChampions website footer. Created by Luna and Artemis. Stats brought to you with help from Legorooj, Nelluk, and the PolyChampions stats team. Graphics done by Newt.<br><br>
    <a href="https://discord.gg/6cvhe4D">Join now on Discord!</a>
```
New line since it's a long line.